vortex-file/src/strategy.rs

```diff
@@ -140,17 +137,9 @@ impl LayoutWriter for BtrBlocksCompressedWriter {
         let chunk = chunk.to_canonical()?.into_array();
 
         // Compute the stats for the chunk prior to compression
-        chunk.statistics().compute_all(STATS_TO_WRITE)?;
```

Comment:
uncompressed_size compute will persist it

Reply:
This line is still present on develop https://github.com/vortex-data/vortex/blob/develop/vortex-array/src/stats/array.rs#L140